apps/next/src/hooks/useOpenApp.ts

```diff
@@ -26,7 +26,10 @@ export const useOpenApp = () => {
     }: {
       closeWindow?: boolean;
       viewMode?: ViewMode;
-      navigateTo?: string;
+      navigateTo?: {
+        path: string;
+        search?: string;
+      };
```

Comment:
At this point maybe it would be better to move this type outside of the function declaration. This will declutter the actual code a bit